docs/SUPPORT.md

```diff
@@ -5,7 +5,7 @@ If you are looking for support for TEAMMATES, check out:
 * User Documentation
   * [For students](https://teammatesv4.appspot.com/web/front/help/student)
   * [For instructors](https://teammatesv4.appspot.com/web/front/help/instructor)
-* [Developer Documentation](../docs/README.md)
+* [Developer Documentation](https://github.com/TEAMMATES/teammates)
```

Comment:
I'd like to amend this to the final dev-docs URL, but I'm not certain what that will be yet